src/SRU/ScanResponse.php

```diff
@@ -5,14 +5,16 @@ class ScanResponse
 {
     public $doc;
 
-    function __construct($xml)
+    /**
+     * @param string|\DOMDocument $xml Scan XML response document
+     * @throws \InvalidArgumentException If $xml is not a string or DOMDocument
+     */
+    public function __construct($xml)
     {
-        if(is_string($xml))
-        {
+        if (is_string($xml)) {
             $this->doc = new \DOMDocument();
             $this->doc->loadXML($xml);
-        } elseif (is_a($xml, '\DOMDocument'))
-        {
+        } elseif (is_a($xml, '\DOMDocument')) {
```

Comment:
No need to change this as it's a linting change, but we would usually do `instanceof` rather than `is_a`